homeassistant/components/vizio/media_player.py

```diff
@@ -276,7 +279,7 @@ async def async_update(self) -> None:
         if self._attr_app_name == NO_APP_RUNNING:
             self._attr_app_name = None
 
-    def _get_additional_app_names(self) -> list[dict[str, Any]]:
+    def _get_additional_app_names(self) -> list[str]:
```

Comment:
I think the type hint here was incorrect

Reply:
You're right, thanks for fixing